driver-sync/src/test/functional/com/mongodb/client/unified/Entities.java

```diff
@@ -497,7 +506,24 @@ private void initClient(final BsonDocument entity, final String id,
         MongoClientSettings clientSettings = clientSettingsBuilder.build();
 
         if (entity.containsKey("observeLogMessages")) {
-            putEntity(id + "-logging-interceptor", new TestLoggingInterceptor(clientSettings.getApplicationName()), clientLoggingInterceptors);
```

Comment:
This change is added in accordance with the [spec](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#entity) `observeLogMessages`